drivers/ethernet/eth_stm32_hal.c

```diff
@@ -1187,6 +1189,103 @@ static int eth_initialize(const struct device *dev)
 	return 0;
 }
 
+static uint32_t reverse(uint32_t val)
+{
+	uint32_t res = 0;
+	int i;
+
+	for (i = 0; i < 32; i++) {
+		if (val & (1 << i)) {
+			res |= 1 << (31 - i);
+		}
+	}
+
+	return res;
+}
+
+static void net_if_mcast_cb(struct net_if *iface,
```

Comment:
`static void net_if_stm32_mcast_cb(struct net_if *iface,`

Reply:
Thanks, I've applied the change.